src/KNXLib/KnxReceiver.cs

```diff
@@ -172,7 +206,16 @@ protected void ProcessCEMI(KnxDatagram datagram, byte[] cemi)
             }
             catch (Exception e)
             {
-                Logger.Error(ClassName, e);

```

Comment:
Hi GioviQ  

This really makes sense!
Thanks!

Can you just change the pull without this Logger.Error changes?
If you check, I added a Logger.Error(string id, Exception e) function that does this (based on your last pulls)